optuna/_callbacks.py

```diff
@@ -0,0 +1,57 @@
+from typing import Tuple
+
+from optuna._experimental import experimental
+from optuna._study_summary import StudySummary  # NOQA
```

Comment:
I think this line is unnecessary.
```suggestion
```

Reply:
Good catch. The `NOQA` stopped it from being caught by the checkers. 😅